text/0000-rust-llm.md

```diff
@@ -0,0 +1,78 @@
+# RFC: Add large language models to Rust
+
+This adds a new module to the standard library containing interfaces to large language models.
+
+The author highlights that no large language models were used in the making of this RFC exactly because Rust does not support them today.
+
+## Motivation
+
+Large language models (LLMs for short) have become ubiquitous.
+But using AI language models in Rust has been a lot harder than one would expect.
+The standard library provides neither an interface to nor an implementation of a large language model.
+This will significantly hurt adoption in the future as Rust continues to fall behind other ecosystems, like NodeJS built-in `node:llm` or C23's `llm.h`.
```

Comment:
other precedence is [Whitespace](https://en.wikipedia.org/wiki/Whitespace_(programming_language)) adding LLM support by defining Line Separator (U+2028) to invoke ChatGPT with the current contents of the stack.